src/main/java/mekanism/client/gui/robit/GuiRobitRepair.java

```diff
@@ -64,7 +63,7 @@ protected void drawGuiContainerForegroundLayer(int mouseX, int mouseY) {
         if (repairContainer.maximumCost > 0) {
             int k = 8453920;
             boolean flag = true;
-            String s = I18n.translateToLocalFormatted("container.repair.cost", repairContainer.maximumCost);
+            String s = LangUtils.localizeWithFormat("container.repair.cost", repairContainer.maximumCost);
```

Comment:
this should use the `I18n` from the client package (there's a bunch of still I still need to do for the lang stuff)

Reply:
Is that the same one I removed, or is it something else, as if I remember it had a deprecation warning (though it might have just been before I started looking at I18n methods to replace them). Also I was debating switching to using TextComponentTranslation for the translation stuff to let the client know it should translate it.

Reply:
`net.minecraft.client.resources.I18n` vs `net.minecraft.util.text.translation.I18n`

`util.text` one doesn't exist 1.13+